metaflow/datastore/content_addressed_store.py

```diff
@@ -76,23 +79,40 @@ def save_blobs(self, blob_iter, raw=False, len_hint=0):
 
         def packing_iter():
             for blob in blob_iter:
+                raw_setting = raw
```

Comment:
logic is to use raw setting but allow each blob to override. Compression methods would be fixed for now (not sure if there is a lot of value in allowing a huge amount of customization here) but we could add another one ourselves.